src/experimental/constructs/error-budget-alarm.ts

```diff
@@ -15,6 +15,16 @@ import { Topic } from "aws-cdk-lib/aws-sns";
 import { Construct } from "constructs";
 import type { GuStack } from "../../constructs/core";
 
+/**
+ * Props for creating an ErrorBudgetAlarm
```

Comment:
Thanks for adding these docs @jorgeazevedo & @michaelwmcnamara 💯

Reply:
Could we move these docs to be inlined to help [the TypeDoc site](https://guardian.github.io/cdk/)?

```ts
export interface ErrorBudgetAlarmProps {
  /**
   * The name of the SLO. Assumed to be unique per stack.
   */
  sloName: string;

  /**
   * The percentange target for the SLO, expressed as a decimal floating point number. Must be between 0.95 and 0.995.
   */
  sloTarget: number;

  badEvents: IMetric;
  validEvents: IMetric;
  snsTopicNameForAlerts: string;
}
```

Reply:
Tangentially related PR: https://github.com/guardian/cdk/pull/1587!